src/Stratis.Bitcoin.Features.PoA/BasePoAFeatureConsensusRules/PoAHeaderSignatureRule.cs

```diff
@@ -93,8 +96,7 @@ public override async Task RunAsync(RuleContext context)
                 if (prevHeader.HashBlock == uint256.Parse("7d67ea42010f03971edd6ba5e1b644d09c9fd0191ca8d312255c12d23f7cd147"))
                     break;
 
-                this.Logger.LogTrace("(-)[TIME_TOO_EARLY]");
-                ConsensusErrors.BlockTimestampTooEarly.Throw();
+                this.Logger.LogWarning($"Block {prevHeader.HashBlock} was mined by the same miner as {blockCounter} blocks ({header.Time - prevHeader.Header.Time})s ago and there was no federation change");
```

Comment:
We could include the miner's public key to see if there's a pattern:

`this.Logger.LogWarning($"Block {prevHeader.HashBlock} was mined by the same miner ('{pubKey.Hex()}') as {blockCounter} blocks ({header.Time - prevHeader.Header.Time})s ago and there was no federation change");`